File: Day20/SwapNumbers.java
package Day20;
import java.util.Scanner;
public class SwapNumbers {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print(("Enter Two number: "));
        int num1 = sc.nextInt();
        int num2 = sc.nextInt();
        sc.close();

        System.out.println("Before swapping: num1 = " + num1 + ", num2 = " + num2);

        int temp;
        temp = num1;
        num1 = num2;
        num2 = temp;

        System.out.println("After swapping: num1 = " + num1 + ", num2 = " + num2);
    }
}

File: Day20/swapNumbers.js
num1 = parseInt(prompt("Enter number 1 : "));
num2 = parseInt(prompt("Enter number 2 : "));

console.log(`Before swapping: num1 = ${num1} , num2 = ${num2}`);

let temp;

temp = num1;
num1 = num2;
num2 = temp;

console.log(`After swapping: num1 = ${num1} , num2 = ${num2}`);
